File: packages/java/src/main/java/com/logolabs/inkvec/InvalidOptionsException.java
package com.logolabs.inkvec;

/**
 * An option is unknown, of the wrong type, or out of range (including a malformed options
 * JSON string, when you pass options as raw JSON).
 */
public final class InvalidOptionsException extends InkvecException {
    private static final long serialVersionUID = 1L;

    InvalidOptionsException(String message) {
        super(message);
    }
}

File: packages/php/src/InvalidOptionsException.php
<?php

declare(strict_types=1);

namespace LogoLabs\Inkvec;

/**
 * An option the tracer does not know, or a value of the wrong type or out of its range
 * (`INKVEC_ERR_INVALID_OPTIONS`). The message names the option.
 */
final class InvalidOptionsException extends InkvecException
{
}
